lib/ServerPilot.php

```diff
@@ -352,10 +394,50 @@ private function _send_request( $url_segs, $params = array(), $http_method = 'ge
 
 		curl_close( $req );
 
-		// Decode JSON by default
-		if( $this->decode )
-			return json_decode( $response );
-		else
-			return $response;
+        // Everything when fine
+        if($http_status == 200)
```

Comment:
To match the rest of the code, use tabs and space in between round brackets e.g. `if( $http_status == 200 )`